notifications_pipeline_steps/NotificationsPipelinePresentation.py

```diff
@@ -0,0 +1,3 @@
+import NotificationsPipelineStep
+class NotificationsPipelinePresentation(NotificationsPipelineStep.NotificationsPipelineStep):
+    pass
```

Comment:
To give an example of how this looks once you remove redundancy in the names and remove full module imports, this will be:

```
from pipeline_step import PipelineStep
class Presentation(PipelineStep):
    pass
```

And the path/name of this file will be: `pipeline_steps/presentation.py`